Lib/Internal.ps1

```diff
@@ -70,7 +70,11 @@ function WriteVerbose {
         [Parameter(Mandatory, ValueFromPipeline)] [System.String] $Message
     )
     process {
-        Write-Verbose -Message ('[{0}] {1}' -f (Get-Date).ToLongTimeString(), $Message);
+        $parentcallstack = (Get-PSCallStack)[1] # store the parent Call Stack        
+        $Functionname = $parentcallstack.FunctionName
+        $LineNo = $parentcallstack.ScriptLineNumber
+        $scriptname = ($parentcallstack.Location -split ':')[0]
```

Comment:
@DexterPOSH Did you intend to push this code up? Presumably you added it to help debug?